go/types/encode_human_readable.go

```diff
@@ -195,6 +204,23 @@ func (w *hrsWriter) writeStruct(v Struct, printStructName bool) {
 	w.write("}")
 }
 
+var dateTimeType = MakeStructTypeFromFields("DateTime", FieldMap{
+	"secSinceEpoch": NumberType,
+})
+
+func (w *hrsWriter) FormatDateTime(v Value) string {
+	if w.tz == nil {
```

Comment:
I see. Another advantage of using a comment is that no flag is necessary. We can just always output the useful string.